CameraOverlay/overlay_all_stats.py

```diff
@@ -129,8 +131,20 @@ def on_message(self, client, userdata, msg):
 				self.data["reed_velocity"] = 0
 				self.data["reed_distance"] = 0
 				self.data["count"] = 0
+				self.data["message"] = 0
+		
+		elif topic == str(topics.DAShboard.recieve_message):
+			message = msg.payload.decode("utf-8")
+
+			# reset counter once message is recieved 
+			self.counter = 0  
 
+			# Display Message
+			self.message_canvas.draw_text("{0}".format(message), (190, self.text_height * 5), size= 1, colour=Colour.red)
+		
+		if self.counter >= 70:
```

Comment:
This only checks to clear the screen whenever a message is received. Also it is tied to the frame rate rather than the time, which will make the timing unstable.